build-ios.sh

```diff
@@ -60,7 +60,8 @@ fi
 CSPROJ_DIR=$(dirname "$CSPROJ_PATH")
 CSPROJ_FILENAME=$(basename "$CSPROJ_PATH")
 
-cd "$CSPROJ_DIR" && cd ../ || exit 1
+NUGET_RESTORE_DIR="${SLN_DIR:-"$CSPROJ_DIR/.."}"
```

Comment:
Thanks for the PR!

Should this line look like this instead to check presence of inputted NUGET_RESTORE_DIR, else fallback?

```
NUGET_RESTORE_DIR="${NUGET_RESTORE_DIR:-"$CSPROJ_DIR/.."}"
```

Reply:
You are absolutely correct! I renamed it from `SLN_DIR` after I realized that was a bit misleading since it's only used for the nuget restore call. Looks like I missed a spot.

Updated and pushed!